Sources/LanguageServerProtocol/Requests/SymbolInfoRequest.swift

```diff
@@ -135,6 +135,32 @@ public struct SymbolDetails: ResponseType, Hashable {
   /// This property only applies to Swift symbols.
   public var systemModule: ModuleInfo?
 
+  /// The line number where the symbol is declared in the file.
+  ///
+  /// This value is 1-based, meaning the first line of the file is numbered 1.
+  /// For example, in the following content:
+  /// ```
+  /// func test() {}
+  /// ```
+  /// If `test()` appears on the second line, `line` would be `2`.
+  public var line: Int?
+
+  /// The length of the symbol’s declaration in characters.
+  ///
+  /// This represents how many characters the symbol’s name and syntax span.
+  /// For example, for the symbol `test()`, `length` would be `6` (including the parentheses `()`).
+  public var length: Int?
+
+  /// The column number where the symbol's declaration starts in the line.
+  ///
+  /// This value is 1-based, meaning the first character of the line is column 1.
+  /// For example, in the following line:
+  /// ```
+  /// func test() {}
+  /// ```
+  /// If `test` starts at the 6th character, the `column` would be `6`.
+  public var column: Int?
+
```

Comment:
Since the `SymbolInfo` request is an LSP extension that  we expose publicly, would should use LSP’s `Position` type (which is UTF-16) instead of sourcekitd’s line, length and column (which is UTF-8). So, I would wrap all of these properties in a `Range<Position>` similar to how it’s returned in `HoverResponse`. And also add the addition to `LSP Extensions.md`.